app/avatar/templates/wallpaper/sunburst.txt

```diff
@@ -11,7 +11,7 @@
                 <polygon id="Triangle" fill="#0FCE7C" transform="translate(227.858016, 195.838178) scale(1, -1) rotate(39.500000) translate(-227.858016, -195.838178) " points="187.090809 -2.22194293 295.413973 278.501822 160.30206 393.898297"></polygon>
                 <polygon id="Triangle" fill="#8E2ABE" transform="translate(182.645562, 253.129569) scale(1, -1) rotate(79.000000) translate(-182.645562, -253.129569) " points="93.9563989 94.4043414 321.394524 411.854796 43.8966013 365.185923"></polygon>
                 <polygon id="Triangle" fill="#F9006C" transform="translate(461.423262, 260.244063) scale(1, -1) rotate(-79.000000) translate(-461.423262, -260.244063) " points="547.019849 101.779771 595.65714 356.480153 327.189383 418.708355"></polygon>
-                <polygon id="Triangle" fill="#3E00FF" transform="translate(414.249183, 200.579420) scale(1, -1) rotate(-39.500000) translate(-414.249183, -200.579420) " points="453.058181 8.14372873 485.079807 393.015084 343.418559 280.33264"></polygon>
+                <polygon id="Triangle" fill="#6F3FF5" transform="translate(414.249183, 200.579420) scale(1, -1) rotate(-39.500000) translate(-414.249183, -200.579420) " points="453.058181 8.14372873 485.079807 393.015084 343.418559 280.33264"></polygon>
```

Comment:
^ I'm assuming this was intentional ?

Reply:
This is swapping the primary colors over - I haven't seen the image though, so I'll switch it back just incase I'm breaking something - there's no reason this can't be blue.

Reply:
Resolved in f9b62ed